content/logs/index.md

```diff
@@ -72,7 +70,7 @@ The Datadog Agent can collect logs from files or the network (TCP or UDP) and fo
 Set `type` to **file** then specify the absolute `path` to the log file you want to tail.
 
 Example: 
-If you want to gather your python app logs for instance stored in **/var/log/myapp1.log** and **/var/log/python.log** you would create a `python.yaml` file as follows:
+If you want to gather your python app logs for instance stored in **/var/log/myapp1.log** and **/var/log/python.log** you would create a `python.d/conf.yaml` file as follows:
```

Comment:
"To gather python applications stored in **/var/log/myapp1.log** and **/var/log/python.log** you would create a `python.d/conf.yaml` file as follows:"

Reply:
Thanks, will change this accordingly.